src/lib/style-properties.html

```diff
@@ -493,6 +494,10 @@
           }
           element._customStyle = style;
         }
+        // @media rules may be stale in IE 10 and 11
```

Comment:
For perf reasons, consider doing this only if styles contain `@media` rules. We should do some perf checking of IE stylesheet parsing to help decide if this is necessary.